docsui/index.ts

```diff
@@ -0,0 +1,24 @@
+import { promises as fs } from "fs"
+
+import { AzureFunction, Context } from "@azure/functions"
+import { getAbsoluteFSPath } from "swagger-ui-dist"
+
+const swaggerJsdocUI: AzureFunction = async function (context: Context): Promise<void> {
+  let filename = context.bindingData.filename
+  filename = filename || "index.html"
+  const body = await fs.readFile(getAbsoluteFSPath() + "/" + filename).then((file) => {
+    if (filename === "swagger-initializer.js") {
```

Comment:
I don't quite get this either. 😅  What is swagger-initializer.js and what is this pet store url. And what is the url we are replacing it with, `/api/Doc`